### PR TITLE
[5.3] Add isRead and unread method to DatabaseNotification

### PR DESCRIPTION
**Pull request by JellyBool**
I found it very useful when display all notifications with view using:

```php
@foreach ($user->notifications as $notification) 
    <div class="notification {{ $notification->isRead ? 'read' : ' ' }}">
       //
    </div>
@endforeach
```

### CODE REVIEW

File: src/Illuminate/Notifications/DatabaseNotification.php [hunk 1]
```diff
@@ -58,6 +58,26 @@ public function markAsRead()
...
+    {
+        return $this->read_at !== null;
+    }
+
+    /**
+     * Determine a notification is not read yet.
+     *
+     * @return bool
+     */
+    public function isNotRead()
```
Comment (GrahamCampbell) [inline src/Illuminate/Notifications/DatabaseNotification.php]: just `notRead` feels better

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Notifications/DatabaseNotification.php (hunk 1 above)
Comment (JellyBool) [inline src/Illuminate/Notifications/DatabaseNotification.php]: OK, I will change that.

**Comment by JellyBool** [comment]
Changed `isNotRead` to `notRead`  @GrahamCampbell

**Comment by taylorotwell** [comment]
Isn't it more common to say "unread"?

**Pull request merged**